node/test/peer_strategies.js

```diff
@@ -207,6 +207,67 @@ allocCluster.test('prefer incoming selects the incoming peer when outgoing is pr
     }
 });
 
+allocCluster.test('subChannel: prefer incoming selects the incoming peer when outgoing is present' , {
+    numPeers: 3,
+    skipEmptyCheck: true,
+    preferConnectionDirection: 'out'
```

Comment:
interesting, how does this allocCluster test option work? I think this should be under something like `channelOptions` if you actually want it, but shouldn't be having an effect as is?